src/components/devtools/TransferIO.tsx

```diff
@@ -128,19 +128,19 @@ function TransferIO() {
                   border: '1px solid var(--text-faded)',
                   padding: '15px',
                 }}
-                placeholder={'Quantity in IO to send'}
-                value={new mIOToken(quantity).toIO().valueOf()}
+                placeholder={'Quantity in ARIO to send'}
+                value={new mARIOToken(quantity).toARIO().valueOf()}
                 setValue={(e) =>
                   setQuantity(
-                    new IOToken(parseInt(e?.toString())).toMIO().valueOf(),
+                    new ARIOToken(parseInt(e?.toString())).toMARIO().valueOf(),
                   )
                 }
                 validationPredicates={{}}
               />
               <span style={{ color: 'var(--accent)' }}>
                 {loading
                   ? 'Loading balance...'
-                  : `Balance: ${formatIO(Math.round(mioToIo(ioBalance)))} IO`}
+                  : `Balance: ${formatIO(Math.round(mioToIo(ioBalance)))} ARIO`}
```

Comment:
should probably use ticker in this file (yes it's dev tools but would still be good)